static/js/main.js

```diff
@@ -50,6 +50,15 @@ const createInputs = () => {
     return inputs;
 };
 
+function populateDefaults(){
+  document.getElementById('input-text-voltage').value = '5';
+  document.getElementById('input-text-length').value = '27';
+  document.getElementById('input-text-r_not').value = '2.3';
+  document.getElementById('input-text-r_a').value = '4.5';
+  document.getElementById('input-text-x').value = '0';
+  document.getElementById('input-text-awg').value = '30';
+  document.getElementById('input-radio-voltage').checked = true;
+}
```

Comment:
In case someone has numbers entered already and they hit the sample input button, can we check for the value of the force input here and clear it if it's non-empty? It seems as though that is the only input in which is not modified. This is re my question from last night's meeting but I understand it's kind of an edge case

Reply:
Just added this!